docs/madr/decisions/038-transition-to-new-policies.md

```diff
@@ -0,0 +1,473 @@
+      minimumHosts: 35
+      threshold: 36
+```
+
+User creates a new MeshCircuitBreaker replacement:
+
+```yaml
+type: MeshCircuitBreaker
+name: backend-inbound-outlier-detection
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+  - targetRef:
+      kind: Mesh
+    default:
+      connectionLimits:
+        maxConnections: 24
+        maxPendingRequests: 25
+        maxRequests: 26
+        maxRetries: 27
+      outlierDetection:
+        interval: 21s
+        baseEjectionTime: 22s
+        maxEjectionPercent: 23
+        splitExternalAndLocalErrors: false
+        detectors:
+          totalFailures:
+            consecutive: 28
+          gatewayFailures:
+            consecutive: 29
+          localOriginFailures:
+            consecutive: 30
+          successRate:
+            requestVolume: 31
+            minimumHosts: 32
+            standardDeviationFactor: 1.33
+          failurePercentage:
+            requestVolume: 34
+            minimumHosts: 35
+            threshold: 36
+```
+
+Using shadow mode we see there is an empty jsonPatches list!
+
+### FaultInjection -> MeshFaultInjection
+
+It should be always safe to remove FaultInjection and create new MeshFaultInjection when necessary. 
+
+### TrafficPermission -> MeshTrafficPermission
+
+```yaml
+type: TrafficPermission
+name: on-redis
+mesh: default
+sources:
+  - match:
+      kuma.io/service: frontend_kuma-demo_svc_8080
+  - match:
+      kuma.io/service: backend_kuma-demo_svc_3001
+destinations:
+  - match:
+      kuma.io/service: redis_kuma-demo_svc_6379
+```
+
+```yaml
+type: MeshTrafficPermission
+name: on-redis
+mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: redis_kuma-demo_svc_6379
+  from:
+    - targetRef:
+        kind: MeshService
+        name: frontend_kuma-demo_svc_8080
+      default:
+        action: Allow
+    - targetRef:
+        kind: MeshService
+        name: backend_kuma-demo_svc_3001
+      default:
+        action: Allow
+```
+
+```yaml
+- value:
+    permissions:
+      - any: true
+    principals:
+      - authenticated:
+          principalName:
+            exact: spiffe://default/backend_kuma-demo_svc_3001
+      - authenticated:
+          principalName:
+            exact: spiffe://default/frontend_kuma-demo_svc_8080
+  op: add
+  path: envoy.config.listener.v3.Listener/inbound:10.42.0.28:6379/filterChains/0/filters/0/typedConfig/rules/policies/MeshTrafficPermission
+- op: remove
+  path: envoy.config.listener.v3.Listener/inbound:10.42.0.28:6379/filterChains/0/filters/0/typedConfig/rules/policies/backend-to-httpbin-4b4zbcc4422zz784
+```
+
+## TrafficRoute -> MeshHTTPRoute(MeshTCPRoute)
+
+Covered by [036-route-transition.md](036-route-transition.md)
+
+## ProxyTemplate -> MeshProxyTemplate
+
+This is where shadow mode shines the most even not in the context of the transition but for a regular usage.
+
+```yaml
+type: MeshProxyPatch
+mesh: default
+name: custom-template-1
+labels:
+  kuma.io/dryrun: yesplease
```

Comment:
Should it be also shadow?

Reply:
yeah, my PoC had this label and I forgot to remove it 😂

Reply:
```suggestion
  kuma.io/effect: shadow
```